src/client/pages/UserPage/components/UserProfile.tsx

```diff
@@ -29,40 +30,26 @@ type Props = InjectedFormProps<UserProfileFormData, UserProfileProps> & UserProf
 
 class UserProfile extends React.Component<Props> {
   genderSelect: React.StatelessComponent<CustomFieldProps> = ({ input, className }) => {
-    const genders = [
-      'Male', 'Female', 'Non-Binary', 'Transgender',
-      'I prefer not to say', 'Other',
-    ];
-
     return (
       <select {...input} className={className}>
-        {genders.map((gender, i) =>
-          <option key={i} value={gender}>{gender}</option>)}
+        {UserGenderOptions.map((UserGenderOptions, i) =>
```

Comment:
You're resetting UserGenderOptions in the first arg of the map here - can we keep it `gender` as it was before?. (note: the previous code is *not* `genders.map((genders, i)...`) which i believe is the source of confusion

Reply:
good catch ty